frontend/app/forms/form-common-options/form-common-options.js

```diff
@@ -59,6 +59,17 @@ export const courtTypeOptions = [
   { label: "Justice Court", value: "Justice" }
 ];
 
+export const previouslyAppliedOptions = [
```

Comment:
I think this should maybe just be a variable inside of the application for pardon web component. I don't know of another form that asks whether the petitioner has previously applied or not.

Additionally, the label and value here are just a Yes/No, which isn't really a list of common options that is specific to this "previously applied" question.